nzpyida/analytics/predictive/classification.py

```diff
@@ -126,15 +126,25 @@ def conf_matrix(self, in_df: IdaDataFrame, id_column: str, target_column: str,
         float
             weighted classification accuracy (WACC)
         """
+
+        params = {
+            'id': id_column,
+            'target': target_column
+        }
+        return self._conf_matrix(in_df, out_matrix_table, params)
+        
+    def _conf_matrix(self, in_df: IdaDataFrame, out_matrix_table: str=None, 
```

Comment:
Well, if we want to make it work for KNN we should make it available for users to pass the parameters (as in predict) inside this method. `conf_matrix` will be the same for decision tree and naive bayes, but in the knn its overwritten because it has few more options

Reply:
ok, I got it